src/awkward/_broadcasting.py

```diff
@@ -605,7 +605,7 @@ def continuation():
                         if mask is None:
                             mask = m
                         else:
-                            mask = backend.index_nplike.bitwise_or(mask, m, out=mask)
+                            mask = backend.index_nplike.logical_or(mask, m, out=mask)
```

Comment:
This might have been a slightly premature change on my part; I had forgotten about the cycle cost for the two instructions. Shall I revert? It's not a huge burden to restore one function.

Reply:
Nevermind: it's not enough to worry about. Here's a test showing that it's less than a factor of 2. Three arrays of 84 kB will fit in my L1d cache (256 kB):

```python
>>> one = np.random.randint(0, 2, 84*1024, np.int8)
>>> two = np.random.randint(0, 2, 84*1024, np.int8)
>>> out = np.empty(84*1024, np.int8)
>>> timeit.timeit("np.logical_or(one, two, out=out)", number=100000, globals=globals())
0.47637764099999913
>>> timeit.timeit("np.bitwise_or(one, two, out=out)", number=100000, globals=globals())
0.267168271002447
```

Either I used to care about factors of 2 or the gap used to be bigger than that.

Reply:
I had a feeling it was about 2-4x, so this is also mildly lower than I'd thought.